string camera_name
float32 period
bool is_spotted
pose_rpy/PoseRPY pose
